src/JustSaying/AwsTools/QueueCreation/SqsBasicConfiguration.cs

```diff
@@ -50,6 +50,12 @@ public void Validate()
                 $"Invalid configuration. {nameof(DeliveryDelay)} must be between {JustSayingConstants.MinimumDeliveryDelay} and {JustSayingConstants.MaximumDeliveryDelay}.");
         }
 
+        if (ServerSideEncryption.KmsDataKeyReusePeriod.TotalSeconds is > 86400 or < 60)
```

Comment:
_Maybe_ would be more readable comparing `KmsDataKeyReusePeriod` to `TimeSpan.FromSeconds(1)` and `TimeSpan.FromHours(24)` (or `TimeSpan.FromDays(1)`)?